### PR TITLE
Add route for listing gists of a user

### PR DESCRIPTION
**Pull request by algomaster99**
Hitting `/gists/user` results in the following response:
res

This is how the [required fields](https://github.com/vega/editor-backend/compare/master...gist-controller?expand=1#diff-0d05ced2b57583ccba243a31e0696631R27) are fetched. It was discussed in the last meeting to use GraphQL instead. Shall I go with it?

### CODE REVIEW

**Comment by domoritz** [comment]
Instead of raw_url, you can also use the gist id (since the start and the user are always the same).

**Comment by algomaster99** [comment]
@domoritz 
There is one more `id` after `/raw/` in URL which helps to fetch content.
Screenshot from 2019-07-07 23-53-50
So I think we will need the `raw_url`.

**Comment by domoritz** [comment]
Is that the revision id?

**Comment by algomaster99** [comment]
@domoritz No I don't think so

**Comment by domoritz** [comment]
It would probably save a few bytes if we figure out what that id is but it's not the most important thing to do so feel free to just use the raw_url.

File: package.json [hunk 1]
```diff
@@ -34,8 +34,10 @@
   "dependencies": {
     "cors": "^2.8.5",
     "dotenv": "^8.0.0",
+    "es6-promise": "^4.2.8",
```
Comment (domoritz) [inline package.json]: Why do we need this.

**Comment by algomaster99** [comment]
@domoritz It is different in the response here and when I manually click the **raw** button. Will work on integrating GraphQL till then.

File: package.json (hunk 1 above)
Comment (algomaster99) [reply package.json]: @domoritz I needed a package called `isomorphic-fetch` for calling the GitHub API. [Their documentation](https://www.npmjs.com/package/isomorphic-fetch) instructed to install this package too.

**Pull request merged**